src/govuk/i18n.mjs

```diff
@@ -15,19 +15,56 @@ function I18n (translations) {
  * returns the appropriate string.
  *
  * @param    {String}  lookupKey  - The lookup key of the string to use.
+ * @param    {Object}  options    - Any options passed with the translation string, e.g: for string interpolation.
  * @returns  {String}             - The appropriate translation string.
  */
-I18n.prototype.t = function (lookupKey) {
+I18n.prototype.t = function (lookupKey, options) {
```

Comment:
Are we calling this `options` to allow for future things to be passed in the object that are _not_ placeholders?

If so, are we happy that we'll have placeholders and other options mixed together in a single object with no real way to differentiate between them? Would we be better off either having a separate arg or having a nested object within options that has the placeholders?

If not, should we call this something else, like `placeholders`?

(I've said placeholders too many times now. It's stopped sounding like a word. I'm not even sure it's the right word. Is there a common term for this that's used by other i18n libraries, and if so should we adopt it?)

Reply:
I noted this in my original spike but I don't think anyone was particularly opinionated on it. A reason to potentially keep it 'as is' is that some things that are passed are kind of both. e.g. The `count` option is both a placeholder AND determines which plural form is used.

Reply:
Happy to stick with `options`, basically because [it's consistent with what i18next are doing](https://www.i18next.com/overview/api#t) 😄